docs/reference/library/math.md

```diff
@@ -99,3 +99,19 @@ $ sum_(i in NN) 1 + i $
 All math functions are part of the `math` [module]($scripting/#modules), which
 is available by default in equations. Outside of equations, they can be accessed
 with the `math.` prefix.
+
+# Accessibility
+To make math accessible, you must provide an alternative description for the
+equation that describes it in natural language. In the future, Typst will
+automatically make equations without alternative descriptions in HTML and PDF
+2.0 export accessible. For more information, see the documentation on the [`alt`
+parameter of `math.equation`]($math.equation.alt) and the [Textual
```

Comment:
there is no further information (basically no information) on the `alt` parameter